### DISCUSSION TITLE
How to not remove uhm aa uh um etc filler words during transcription

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [openai/whisper#949] [Q&A]
Can we do that? What parameter it requires?

**Comment by mayeaux** [comment]
You can just write a script to remove the words you want from the text documents after they're transcribed

**Reply by FurkanGozukara** [reply]
i just noticed title was incorrect. i was looking for not to remove :D

**Comment by glangford** [comment]
See
* https://github.com/openai/whisper/discussions/855

and the other discussions referred to there.

**Comment by LaurinmyReha** [comment]
https://github.com/nyrahealth/CrisperWhisper

This should solve your problems
